requirements/base.txt

```diff
@@ -0,0 +1,2 @@
+aiohttp<2.3
```

Comment:
No I won't expect it work on any other version than 2.2.x. Maybe it work on master. But what should I choose?